graspologic/embed/n2v.py

```diff
@@ -57,6 +57,11 @@ def node2vec_embed(
         number of walks as a high degree node (which it will if this setting is not on), then the lower degree nodes
         will take a smaller breadth of random walks when compared to the high degree nodes. This will result in your
         lower degree walks dominating your higher degree nodes.
+    random_seed : int
+        Seed to be used for reproducible results. Default is None and will produce a random output. Note that for a fully
+        deterministically-reproducible run, you must also limit to a single worker thread (`workers=1`), to eliminate
+        ordering jitter from OS thread scheduling. (In Python 3, reproducibility between interpreter launches also requires
```

Comment:
Since we're Python 3.6 plus, we should elevate this from a parenthetical to the top level.  Maybe `OS thread scheduling. In addition, you must set the environment variable \`\`PYTHONHASHSEED\`\` environment variable to control hash randomization.`

Make sure you use the double \`\` there - Sphinx will treat a single \` as something special and then complain

Reply:
Done